tests/collectioncardmakertest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+require('../src/collectioncardmaker.php');
+
+use PHPUnit\Framework\TestCase;
+
+class CollectionCardMakerTest extends TestCase {
+
+
+public function testcreateGameCard()
+{
+   
+    $name = 1;
```

Comment:
Do you want to change this back to a string to reflect an accurate success test?